lib/Service/RoomService.php

```diff
@@ -547,29 +547,6 @@ public function setMessageExpiration(Room $room, Participant $participant, int $
 			->where($update->expr()->eq('id', $update->createNamedParameter($room->getId(), IQueryBuilder::PARAM_INT)));
 		$update->executeStatement();
 
-		$room->setMessageExpiration($seconds);
```

Comment:
This one line should be kept?!🤔

Reply:
No more.

Reply:
Not to context of this method, but, we can maintain  if we will need the newest value in other methods.

Reply:
Well the server's response should have the new value, so yeah the object should be updated.

Reply:
Ah this should go into #7595 
Or should we close that and just merge this one?